### DISCUSSION TITLE
backup:run command is not working in controller

### DISCUSSION THREAD

**Discussion by creativitycomkw** [spatie/laravel-backup#1523] [Q&A]
`<?php

namespace App\Http\Controllers\Admin;

use App\Http\Controllers\Controller;
use Illuminate\Http\Request;
use Illuminate\Support\Facades\Artisan;

class BackupController extends Controller
{
    public function index()
    {
        //
    }

    public function generate(Request $request)
    {
        Artisan::call('backup:run', ['--only-db' => true]);

        return redirect()->back()->with('status', 'Backup generated successfully!');
    }
}
`

**Comment by erikn69** [comment]
https://github.com/spatie/laravel-backup/discussions/1520#discussioncomment-2528858
